### PR TITLE
1、modify ineffassign warning and add ineffassign to makefile

### PR DESCRIPTION
**Pull request by lyh169**
2、add vet

### CODE REVIEW

File: wallet/wallet_proc.go [hunk 1]
```diff
@@ -1177,7 +1177,7 @@ func (wallet *Wallet) SaveSeed(password string, seed string) (bool, error) {
 func (wallet *Wallet) saveSeed(password string, seed string) (bool, error) {
 
 	//首先需要判断钱包是否已经设置seed，如果已经设置提示不需要再设置，一个钱包只能保存一个seed
-	exit, err := wallet.walletStore.HasSeed()
+	exit, _ := wallet.walletStore.HasSeed()
```
Comment (33cn) [inline wallet/wallet_proc.go]: 全部改成asset.Nil(t, err)

Comment (33cn) [reply wallet/wallet_proc.go]: @lyh169

Comment (lyh169) [reply wallet/wallet_proc.go]: haode

**Comment by vipwzw** [comment]
#46

**Pull request merged**